metadata-service/openapi-servlet/src/main/java/io/datahubproject/openapi/v3/OpenAPIV3Generator.java

```diff
@@ -570,19 +570,15 @@ private static void addExtraParameters(final Components components) {
         "SortBy" + MODEL_VERSION,
         new Parameter()
             .in(NAME_QUERY)
-            .name("sort")
+            .name("sortCriteria")
```

Comment:
need a note in the release notes section for this

Reply:
breaking change

Reply:
Yes, the API controller still supports "sort", however client sdks using the spec will break the programmatic apis.

Reply:
It technically doesn't remove the deprecated parameter so if you were previously sending in a single string here _it would not break with these changes_. It only affects the documentation and OpenAPI spec. Will add a note nonetheless.